pvlib/modelchain.py

```diff
@@ -860,8 +861,12 @@ def prepare_inputs(self, weather, times=None):
 
         self.times = self.weather.index
 
+        kwargs = _build_kwargs(['pressure', 'temp_air'], weather)
+        kwargs['temperature'] = kwargs.pop('temp_air')
```

Comment:
this raises a `KeyError` when `temp_air` does not exist. Use `try: ... except KeyError: pass`